check50/__main__.py

```diff
@@ -278,87 +313,97 @@ def main():
     global SLUG
     SLUG = args.slug
 
-
+    # dev implies offline and verbose "info" if not overwritten
     if args.dev:
         args.offline = True
-        args.verbose = True
+        if not args.verbose:
+            args.verbose = "info"
 
+    # offline implies local
     if args.offline:
+        args.no_install_dependencies = True
+        args.no_download_checks = True
         args.local = True
 
+    # Setting any verbosity level implies logs from checks
     if args.verbose:
-        # Show lib50 commands being run in verbose mode
-        logging.basicConfig(level=os.environ.get("CHECK50_LOGLEVEL", "INFO"))
-        lib50.ProgressBar.DISABLED = True
         args.log = True
 
+    # Setup logging for lib50 depending on verbosity level
+    setup_logging(args.verbose)
+
+    # Warning in case of running remotely with no_download_checks or no_install_dependencies set
+    if (args.no_download_checks or args.no_install_dependencies) and not args.local:
+        if args.no_download_checks and args.no_install_dependencies:
+            msg = _("--no-downloads-checks and --no-install-dependencies have")
+        elif args.no_download_checks:
+            msg = _("--no-downloads-checks has")
+        else:
+            msg = _("--no-install-dependencies has")
+        termcolor.cprint(_("Warning: {} no effect without --local").format(msg), "yellow", attrs=["bold"])
```

Comment:
Instead of repeating `args.no_*` twice and handling every possible combination separately, would the following be logically equivalent and more scalable perhaps or would it cause a problem with `gettext`?

```
if not args.local:
  invalid_args = []
  if args.no_download_checks:
    invalid_args.append(_("--no-download-checks"))
  
  if args.no_install_dependencies:
    invalid_args.append(_("--no-install-dependencies"))

  if invalid_args:
    termcolor.cprint(_("Warning: {} no effect without --local").format(_("and").join(invalid_args), _("have") if len(invalid_args) > 1 else _("has")), "yellow", attrs=["bold"])
```

Reply:
My solution is already not great with `gettext` as Chad mentioned in the previous PR, but this looks rather undo-able. From my understanding, each occurrence of `_()` results in another sentence that needs translating. So `_(have)` would be a sentence on its own. My knowledge of Spanish is roughly zero, si, but I worry this might be untranslatable as the location of the words be different in a Spanish sentence?

Anyway, I don't think is much of a concern as we shouldn't be scaling our `--offline` flags any further. What's next, `--no-update-checks`?

Reply:
Actually I don't think it makes sense to translate things like `--no-download-checks` and `--local`? And we could eliminate the "has" and "have" altogether by changing the message to something like:

```
Warning: you should always use --local when using --no-download-checks, ...
```

Just like Docker does with the `--memory-swap` option or something?

Reply:
done, thanks!